<ui version="4.0" >
 <class>MWDiag</class>
 <widget class="QWidget" name="MWDiag" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="lbIFile" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Hard Mask File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leIFile" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="valInmsk" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>170</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>170</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>118</red>
             <green>116</green>
             <blue>108</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text" >
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbIFile" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="lbOFile" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Output Mask    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOFile" />
     </item>
     <item>
      <widget class="QPushButton" name="pbOFile" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="lbSignificance" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Errorbar Criterion</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSignificance" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>63</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_15" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckAngles" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>No Solid Angles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbIndividual" >
     <property name="title" >
      <string>Individual White Beam Tests</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="MantidQt::MantidWidgets::MWRunFiles" name="white_file" >
        <property name="label" stdset="0" >
         <string>White Beam Van 1</string>
        </property>
        <property name="multipleFiles" stdset="0" >
         <bool>false</bool>
        </property>
        <property name="optional" stdset="0" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="lbHighAbs" >
          <property name="text" >
           <string>High counts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leHighAbs" />
        </item>
        <item>
         <widget class="QLabel" name="lbLowAbs" >
          <property name="text" >
           <string>Low counts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLowAbs" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <widget class="QLabel" name="lbHighMed" >
          <property name="text" >
           <string>Median test high</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leHighMed" />
        </item>
        <item>
         <widget class="QLabel" name="lbLowMed" >
          <property name="text" >
           <string>Median test low</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLowMed" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbVariation" >
     <property name="title" >
      <string>Efficiency Variation Test</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="MantidQt::MantidWidgets::MWRunFiles" name="white_file_2" >
        <property name="label" stdset="0" >
         <string>White Beam Van 2</string>
        </property>
        <property name="multipleFiles" stdset="0" >
         <bool>false</bool>
        </property>
        <property name="optional" stdset="0" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="lbVariation" >
          <property name="text" >
           <string>Proportional change criterion</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leVariation" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBackTest" >
     <property name="title" >
      <string>Background check</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="ckDoBack" >
        <property name="text" >
         <string>Check backgrounds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2" >
       <widget class="QLabel" name="lbAcceptance" >
        <property name="text" >
         <string>Acceptance Factor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLineEdit" name="leAcceptance" />
      </item>
      <item row="0" column="5" >
       <widget class="QLabel" name="validator_label_3" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lbStartTime" >
        <property name="text" >
         <string>Start TOF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="leStartTime" />
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="lbEndTime" >
        <property name="text" >
         <string>End TOF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLineEdit" name="leEndTime" />
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QCheckBox" name="ckZeroCounts" >
        <property name="text" >
         <string>Reject zero background</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bleed_group" >
     <property name="title" >
      <string>PSD Bleed</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="toolTip" >
         <string>Set the maximum allowed rate per frame for a tube</string>
        </property>
        <property name="text" >
         <string>Max framerate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="bleed_maxrate" >
        <property name="toolTip" >
         <string>Set the maximum allowed rate per frame for a tube</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_2" >
        <property name="toolTip" >
         <string>Set the number of central pixels to ignore within a tube</string>
        </property>
        <property name="text" >
         <string>Ignored pixels</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLineEdit" name="ignored_pixels" >
        <property name="toolTip" >
         <string>Set the number of central pixels to ignore within a tube</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
